features/claims/litigator/litigator_trial_claim_draft_submit.feature

```diff
@@ -33,11 +34,23 @@ Feature: Litigator partially fills out a draft final fee claim, then later edits
 
     Then I click "Continue" in the claim form
 
-    And I enter defendant, representation order and MAAT reference
-    And I add another defendant, representation order and MAAT reference
+    And I enter defendant, post agfs reform representation order and MAAT reference
+    And I add another defendant, post agfs reform representation order and MAAT reference
 
```

Comment:
As you've added `scheme 10` to the [check below](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2434/files#diff-51c02c9b2dcbb821dadbd01c1cf9656bR26) this would be clearer id updated too.

Especially as scheme 11 will happen eventually!

Reply:
are 

`And I enter defendant, post agfs reform representation order and MAAT reference` 

and 

`And I enter defendant, scheme 10 representation order and MAAT reference` 

the same? If so consider using only one of them for DRYness.